### ISSUE TITLE
EI: oddities in item descriptions

### ISSUE THREAD

**Issue by CelticMinstrel** [wesnoth/wesnoth#9070]
There are likely more than just this, I just happened to notice these when someone commented on another issue.

1. The text at the bottom should probably read something like this:

_This item **CANNOT** be dropped, and **CANNOT** be reused_

2. Also, it's very weird to have a hyphen in "hit points".

3. And the word "reused" doesn't really fit there, as that implies it's being used a second time, but if the unit dies it never had a chance to be used at all.

4. Also, button labels are not sentences. They should not end in a period. I don't think there's any reason to include the pronoun, either – just "Take" or "Leave" should be sufficient.

![](https://private-user-images.githubusercontent.com/13679322/345997139-c0f770b0-6456-4720-b8c3-f605889aef0b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAyNzI0MDAsIm5iZiI6MTcyMDI3MjEwMCwicGF0aCI6Ii8xMzY3OTMyMi8zNDU5OTcxMzktYzBmNzcwYjAtNjQ1Ni00NzIwLWI4YzMtZjYwNTg4OWFlZjBiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA2VDEzMjE0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI5NDZkMjgzNTFiMGQ3Yzk4YjdiMDc1OWJkYmI4NThkYjUzYmFlNjE4YTVkYThmYTA5YTZjYmQyMzgzMTlkYWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.u6g2ZRz4ws7PhQI6a4H0OqI0lop-VhcUUNJYq-25KNI)

**Comment by Wedge009** [comment]
I'm guessing some of these are on me, but I'm not really sure what to do with them. I didn't quite agree with the use of the word 'droppable', but I couldn't think of anything as succinct, so perhaps I'll have to revert to that if you object that strongly to it. I'm pretty sure Dalas has it as 'hitpoints' in the original text.

Pretty sure the 'Take It.' + 'Leave It.' labels for the buttons wasn't me though. And I agree with cannot vs can not, but I've had too many reason that can not is an acceptable form.

**Comment by CelticMinstrel** [comment]
> I'm pretty sure Dalas has it as 'hitpoints' in the original text.

I think I would probably prefer it to be written as 'hit points'.

> Reused makes sense in the context that another unit can pick up the item that was dropped on the original user's death.

I'm saying that doesn't really make sense, because if another unit is doing that, it implies the item was never used by the original unit… though, if the item is something like an equippable passive bonus, then I guess it does fit.

**Comment by Wedge009** [comment]
I'm not sure why you think the item is not used by the original unit. Have you played the current EI campaign? It makes sense to me, having seen the reusable item mechanic.

**Comment by CelticMinstrel** [comment]
I haven't played EI. I'm reacting entirely to your screenshot of that one item.

If the idea is that taking the potion means it'll be drunk immediately, then I guess the "reuse" wording does make sense.

**Comment by Dalas121** [comment]
1) I have no opinion on "CANNOT" vs "can NOT"; either one is fine by me.  Not sure it's worth retranslating the strings for something this minor though.

2) Wedge is correct that the original read "hitpoints". I'm fine with any wording, though again, I'm not sure it's significant enough to be worth retranslating.

3) EI has a number of items. Most can be dropped at will or when the user dies, but not all:

`#define NOTE_REUSEABLE
    _"<span size='small'><i>  This item is reusable if its wielder dies, and can be dropped by right clicking.</i></span>
<span size='small'><i>  Picking up an item consumes your moves and attack.</i></span>
<b></b>"
#enddef
#define NOTE_SINGLE_USE
    _"<span size='small'><i>This item can <b>NOT</b> be dropped, and can <b>NOT</b> be reused if its user dies.</i></span>
<span size='small'><i>  Picking up an item consumes your moves and attack.</i></span>
<b></b>"
#enddef`

4) I have no strong opinion on "Take it." vs "Take", but AFAIK "Take it." is considered a complete sentence: https://www.grammarly.com/blog/imperative-sentences/ 

If you want to make a PR tweaking any of these points, I'll be happy to approve it.

**Comment by Wedge009** [comment]
1. In my experience, 'cannot' is the preferred form (unless 'can' and 'not' are really meant to be separate words which can happen in circumstances outside of the context we're using here). I think it's worth changing for master, probably won't be accepted for 1.18.
2. Nothing for you to do here.
3. I think the context here is useful for celmin. Most items (that I encountered, anyway) will have the reusable note, it was only the Potion in my screenshot that had the single-use wording. Which makes sense, given it's a potion. But most other items (at least the ones that I found) can be dropped and reused by another unit if desired. Handy in the Captured scenario where confiscated items may be recovered by units that didn't have it originally (and you wish for the original user to take possession again, for tactical reasons).
